test/integration/common_defs.sh

```diff
@@ -1,7 +1,7 @@
 set -e
 CPODIR=/tmp/___cpo_integ_test
 rm -rf ${CPODIR}
-EPS=("tcp+k2rpc://0.0.0.0:10000" "tcp+k2rpc://0.0.0.0:10001" "tcp+k2rpc://0.0.0.0:10002")
+EPS=("tcp+k2rpc://0.0.0.0:10000" "tcp+k2rpc://0.0.0.0:10001" "tcp+k2rpc://0.0.0.0:10002" "tcp+k2rpc://0.0.0.0:10003")
```

Comment:
why do you need a fourth endpoint?

Reply:
Without it, cpo gives no node available error when I try to create a collection as all three nodes are used by existing `HTTPClient` collection created by HTTPProxy

Reply:
If you are adding the create collection API as part of this PR then I think you should remove the default collection and schemas that the HTTPProxy creates.

Reply:
Done. Deleted code to create default collection/schema from HTTPProxy class and moved the code to python test. Changed node number to 2 for default collection.